src/common/config.js

```diff
@@ -72,7 +72,7 @@ const Config = {
   configApiUrl: `${baseURL}${configPath}/v1/`,
   telemetryApiUrl: `${baseURL}${telemetryPath}/v1/`,
   authApiUrl: `${baseURL}${authPath}/v1/`,
-  msTelemetryApiRefreshInterval: 2
+  msTelemetryApiRefreshInterval: 15
```

Comment:
is this second or millisecond may be rename it to telemetryApiRefreshIntervalInSeconds